src/shared/components/navbar/Navbar.tsx

```diff
@@ -37,6 +38,12 @@ const Navbar = () => {
           {IMPORTANT_LINKS.map(link => (
             <NavLink key={link.to} activeClassName={csx.activeLink} {...link} />
           ))}
+          <Guard.Unprotected>
```

Comment:
This should be done via simple filter function - don't change implementation. Check for reference getLinksByAuthState in current master branch.

Reply:
done